CSI_Greenville_HIST8510_Final_Project-AR.qmd

```diff
@@ -0,0 +1,390 @@
+
+by.decade.horry <- CSI.HORRY.TIDY %>%
+  arrange(YEAR) %>%
+  mutate(DECADE = if_else(YEAR >= 1800 & YEAR < 1810, "Aughts",
+                          if_else(YEAR >= 1810 & YEAR < 1820, "Tens",
+                          if_else(YEAR >= 1820 & YEAR < 1830, "Twenties",
+                          if_else(YEAR >= 1830 & YEAR < 1840, "Thirties",
+                          if_else(YEAR >= 1840 & YEAR < 1850, "Forties",
+                          if_else(YEAR >= 1850 & YEAR < 1860, "Fifties",
+                          if_else(YEAR >= 1860 & YEAR < 1870, "Sixties",
+                          if_else(YEAR >= 1870 & YEAR < 1880, "Seventies",
+                          if_else(YEAR >= 1880 & YEAR < 1890, "Eighties",
+                          if_else(YEAR >= 1890 & YEAR < 1900, "Nineties",                                   "Other"))))))))))) %>% as_tibble()
+```
+
+```{r}
+#chart Horry by DEATH.TYPE
+death.type.horry <- CSI.HORRY.TIDY %>%
+  group_by(DEATH.TYPE) %>%
+  summarize(count = n())
+  
+ggplot(death.type.horry, aes(x = DEATH.TYPE, y = count)) +  geom_col() + labs (title = "Horry Death Ranked")
+```
+
+```{r}
+#chart DEATH.TYPE by DECADE
+
+decade.count.horry <- by.decade.horry %>%
+  arrange(YEAR) %>%
+  group_by(DECADE, DEATH.TYPE) %>%
+  summarize(count = n())
+
+ggplot(decade.count.horry, aes(DEATH.TYPE, count)) + theme(axis.text.x=element_text(angle=90,hjust=1)) + geom_col() + facet_wrap(~DECADE) + labs (title = "Horry Death by Decade")
+```
+
+As points of comparison emerge from the data it is important to remember the two contributing factors to the recorded data gleaned from the coroners inquests. First, the state of public health that the inquests represent. Many cultural realities affect which people and deaths were examined by a coroner in 19th century South Carolina. For example, a woman who died during childbirth would most likely not have warranted a coroner's report. Second, how recent the advent of the recording of death was in the 19th century. The mortality census is the first time that the United States attempted to keep track of the people and mode of death of its residents. Berry argues that the ability to quantify death, although flawed, contributed to the doubling of the life expectancy between 1850-1950.
+
+```{r}
+#combines all tidy datasets for Greenville, Chesterfield, Anderson and Horry Counties. The reason I chose these three to compare to Greenville is geographical (from other areas of the state) and well as number of records available (Anderson is similar in size to Greenville, Chesterfield is one of the largest datasets and Horry the smallest)
+COMBINED_CSI_TIDY <- rbind(CSI.GVL.TIDY, CSI.ANDERSON.TIDY, CSI.CHESTERFIELD.TIDY, CSI.HORRY.TIDY)
+```
+
+```{r}
+#shows the year with the largest number of inquests from each county
+highest.year <- COMBINED_CSI_TIDY %>%
+  group_by(COUNTY, YEAR) %>%
+  summarize(count = n()) %>%
+  slice_max(count)
+
+highest.year
```

Comment:
Remember, when you run a variable like this it prints out a dataframe to your .qmd document. In this case there are only 4 rows but in some cases further down in your paper not only does the output get cut off but it creates an awkward reading experience. I think you don't actually need to print each of these variable and could either just discuss the results and/or create a chart to visualize it. In instances where you want to discuss the nuances of the data - there are ways to generate a more readable table in a .qmd document. https://quarto.org/docs/blog/posts/2022-02-15-feature-tables/